Sources/DistributedActors/ActorSystem.swift

```diff
@@ -100,20 +100,37 @@ public final class ActorSystem: _Distributed.ActorTransport, @unchecked Sendable
     // ==== ----------------------------------------------------------------------------------------------------------------
     // MARK: Metrics
 
-    // TODO: Use ManagedAtomicLazyReference to store this
-    private lazy var _metrics: ActorSystemMetrics = ActorSystemMetrics(self.settings.metrics)
-    internal var metrics: ActorSystemMetrics {
-        self._metrics
-    }
+    internal let metrics: ActorSystemMetrics
 
     // ==== ----------------------------------------------------------------------------------------------------------------
     // MARK: Cluster
+    internal final class Box<T> {
+        var value: T
+        init(_ value: T) {
+            self.value = value
+        }
+    }
 
     // initialized during startup
     // TODO: Use ManagedAtomicLazyReference to store this
     internal var _cluster: ClusterShell?
```

Comment:
That's another one we'd love to be not the variable anymore, do you want to take it up?